src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/BehandlingInfo.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { BodyShort, HGrid, HStack, VStack } from '@navikt/ds-react';
+
+import { useApp } from '../../../../../context/AppContext';
+import { useVilkårperioder } from '../../../../../hooks/useVilkårperioder';
+import DataViewer from '../../../../../komponenter/DataViewer';
+import { VilkårsresultatIkon } from '../../../../../komponenter/Ikoner/Vurderingsresultat/VilkårsresultatIkon';
+import { BehandlingFaktaTilsynBarn } from '../../../../../typer/behandling/behandlingFakta/behandlingFakta';
+import { byggHenterRessurs, byggTomRessurs, Ressurs } from '../../../../../typer/ressurs';
+import { formaterNullablePeriode } from '../../../../../utils/dato';
+import { aktivitetTypeTilTekst } from '../../../Inngangsvilkår/Aktivitet/utilsAktivitet';
+import { målgruppeTypeTilTekst } from '../../../Inngangsvilkår/typer/vilkårperiode/målgruppe';
+import { VilkårPeriodeResultat } from '../../../Inngangsvilkår/typer/vilkårperiode/vilkårperiode';
+import { Vilkårsresultat, Vilkårsvurdering } from '../../../vilkår';
+
+const StyledHGrid = styled(HGrid)<{ bottomBorder?: boolean }>`
+    padding-bottom: 1rem;
+    ${(props) => props.bottomBorder && `border-bottom: solid 1px white;`}
+`;
+
+export const BehandlingInfo: React.FC<{ behandlingId: string }> = ({ behandlingId }) => {
+    const { request } = useApp();
+    const { vilkårperioderResponse } = useVilkårperioder(behandlingId);
+
+    const [vilkårsvurdering, settVilkårsvurdering] =
+        useState<Ressurs<Vilkårsvurdering>>(byggTomRessurs());
+
+    const hentVilkårsvurdering = useCallback(() => {
+        settVilkårsvurdering(byggHenterRessurs());
+        return request<Vilkårsvurdering, void>(`/api/sak/vilkar/${behandlingId}`).then(
```

Comment:
Tror vi har brukt null en det steder for request som ikke sender noe 👀 
```suggestion
        return request<Vilkårsvurdering, null>(`/api/sak/vilkar/${behandlingId}`).then(
```